channels.yaml

```diff
@@ -8,10 +8,10 @@ releases:
   - version: v1.19.16+k3s1
     minChannelServerVersion: v2.6.0-alpha1
     maxChannelServerVersion: v2.6.99
-  - version: v1.20.13+k3s1
+  - version: v1.20.14-rc1+k3s1
     minChannelServerVersion: v2.6.0-alpha1
     maxChannelServerVersion: v2.6.99
-  - version: v1.21.4+k3s1
+  - version: v1.21.7+k3s1
```

Comment:
I don't think we need this change since we have it on line 128 and rancher wants to have all the versions.